### PR TITLE
Fixes typo in project website URL

### PR DESCRIPTION
**Pull request by christophgockel**

### CODE REVIEW

**Comment by felipesere** [comment]
:+1: LGTM

**Comment by ecomba** [comment]
Yup, go ahead. Looks fine...

**Comment by dirv** [comment]
I think we need one more upvote first.

File: theme/index.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
   </div>
   <div id="footer">
     <p>&copy; 2011-2015 Daniel Irvine.</p>
-    <p>This site is proudly generated by <a href="http://www.pinalj.com/">Pinaclj</a>.</p>
+    <p>This site is proudly generated by <a href="http://www.pinaclj.com/">Pinaclj</a>.</p>
```
Comment (ecomba) [inline theme/index.html]: This is nice! I like how you made that 'j' work with the name of the link! :+1:

**Comment by heruku** [comment]
I'll review when I get home

**Comment by dirv** [comment]
:-1: it should actually be https://www.github.com/dirv/pinaclj

**Comment by ecomba** [comment]
Why so secure @dirv ? Is there a reason for that?

**Comment by felipesere** [comment]
I am not sure if the link is webscale...

**Comment by ecomba** [comment]
Has this been deployed to live already?

**Comment by christophgockel** [comment]
@dirv updated the URL

**Comment by dirv** [comment]
@christophgockel How about removing the `www.`?

**Pull request merged**